lib/service/Service.ts

```diff
@@ -356,6 +356,12 @@ class Service extends EventEmitter {
    * Subscribe to orders being added to the order book.
    */
   public subscribeAddedOrders = (callback: (order: Order) => void) => {
+    this.orderBook.pairIds.forEach((pair) => {
+      const ownOrders = this.orderBook.getOwnOrders(pair);
+      const peerOrders = this.orderBook.getPeersOrders(pair);
+      const orders = [...ownOrders.buy, ...ownOrders.sell, ...peerOrders.buy, ...peerOrders.sell];
+      orders.forEach(order => callback(order));
```

Comment:
I believe this can just be written as `orders.forEach(callback). And I actually am realizing the same can be done for the same lines below ('callback' instead of 'order => callback(order)`) and I'm pretty sure that was written by me. It's only a style thing, but might as well have the newly added lines be less verbose.